tools/project_manager/show_good_word_suggestions_detail.php

```diff
@@ -136,11 +131,8 @@
 echo "</div>";
 
 echo "<div class='overflow-hidden display-flex' style='flex-direction: column;'>";
-if (isset($imagefile)) {
-    echo "<div id='page-browser' class='overflow-hidden'></div>";
-} else {
-    echo "<p style='margin: 0.5em'>" . _("Select one of the page links to view the page image (scan).") . "</p>";
-}
+echo "<iframe name='imageframe' style='border: none;' title='Image' width='100%' height='100%' srcdoc='<p>" . _("Select one of the page links to view the page image (scan).") . "</p>'></iframe>";
```

Comment:
We shouldn't have to use an iframe here -- we need to be able to embed the PageBrowser in other pages as a reusable widget.

Reply:
I'll see what I can do.